Svom Missionpage (#2860)
nasa-gcn/gcn.nasa.gov: app/routes/missions.svom/route.mdx

---
handle:
  breadcrumb: SVOM
---

import logo from './logo_svom_black.svg'

# Space-based multi-band astronomical Variable Objects Monitor (SVOM)

<img
  src={logo}
  align="right"
  width="818"
  height="780"
  alt="SVOM logo"
  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
/>

**Launch Date:** June 22, 2024

**Anticipated Start of Science Operations:** February 2025

**End of Operations:** Mission duration of 3 years

**Notices Archive:** [SVOM French Science Center](https://fsc.svom.org/alerts)

> The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://cnes.fr/en/projects/svom). SVOM mission is dedicated to the study of the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).

Reviewer: ```suggestion
The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://cnes.fr/en/projects/svom). SVOM is dedicated to the study of the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).
```

 <div className="overflow-table">

| Instruments | Energy Range                                           | Field of View | Localization                               |
| ----------- | ------------------------------------------------------ | ------------- | ------------------------------------------ |
| ECLAIRs     | 4 - 250 keV                                            | 2 ster        | ~10ʹ radius (statistical + systematic)     |
| GRM         | 15 - 5000 keV                                          | 2.6 ster      | ~10 degrees                                |
| MXT         | 0.2 - 10 keV                                           | 1.1° x 1.1°   | ~10-100ʺ radius (statistical + systematic) |
| VT          | 450 - 650 nm (Blue band) <br/>650 - 1000 nm (Red band) | 26ʹ x 26'     | ~1ʺ radius                                 |

 </div>

#### GCN Kafka topics

SVOM distributes alerts and localizations of gamma-ray burst and other transients. For now, these notices are published only in VOEvent format through the GCN Kafka topics prefixed `gcn.notices.svom.voevent`.

> Each of the 4 instruments have their dedicated topics to stream notices with their related information, see table below:

Reviewer 2: ```suggestion
Each of the four instruments has its own dedicated topic to stream notices with their related information, see table below:
```

> Each of the four instruments has its own dedicated topic to stream notices with their related information, see table below:

Reviewer: ```suggestion
Each of the four instruments has its own dedicated topic to stream notices with their related information. See table below:
```

| ---------------------------------- | ----------------------------------------------------------------------- |
| `gcn.notices.svom.voevent.grm`     | `grm-trigger`                                                           |
| `gcn.notices.svom.voevent.eclairs` | `eclairs-wakeup` <br/>`eclairs-catalog`<br/ > `slewing` / `not-slewing` |
| `gcn.notices.svom.voevent.mxt`     | `mxt-initial` <br/>`mxt-update`                                         |
| `gcn.notices.svom.voevent.vt`      | <b>Available soon</b>                                                   |

 </div>

#### VOEvent XML-serialized GCN notice types

 <div className="overflow-table">

| Notice Type                                | Content                                                            | Latency       |
| ------------------------------------------ | ------------------------------------------------------------------ | ------------- |
| `grm-trigger`                              | GRM trigger notice with possible rough localization                | ~15 sec       |
| `eclairs-wakeup`                           | ECLAIRs localisation of a GRB candidate                            | ~15 sec       |

> | `eclairs-catalog`                          | ECLAIRs localisation of a flaring source of the on-board catalogue | ~15 sec       |

Reviewer 2: ```suggestion
| `eclairs-catalog`                          | ECLAIRs localisation of a flaring source from the on-board catalogue | ~15 sec       |
```

| `slewing` /<br/>`not-slewing` <sup>1</sup> | Platform slew status and target coordinates                        | ~1-8&nbsp;min |
| <pre>`mxt-initial_qf#`<sup>2</sup></pre>   | MXT localisation of a found counterpart (only if platform slews)   | ~3 min        |
| <pre>`mxt-update_qf#`<sup>2</sup></pre>    | MXT updated localisation (qf > mxt-initial_qf)                     | ~3 min        |

 </div>

<sup>1</sup> `not-slewing`are published either if the detection did not reach
the slew threshold (in a maximum time window of 8 min) or if the slew was not
possible. The reason is specified in the notice parameters.

<sup>2</sup> the value of the quality factor `qf` indicates the accuracy
of the localisation:
 - `qf1`&#8594; 120'' < R90 < 300''<br/>
 - `qf2`&#8594; 60'' < R90 < 120''<br/>
 - `qf3`&#8594; 30'' < R90 < 60''<br/>
 - `qf4`&#8594; R90 < 30''<br/>

#### VOEvent examples

The complete list of published notices can be found [here](https://fsc.svom.org/alerts). Here are 3 different examples of SVOM VOEvent notices:

<details>
    <summary>`eclairs-wakeup`</summary>

>     ``` xml

Reviewer: ```suggestion
    ```xml
```

          <ObservatoryLocation id="GEOLEO"/>
          <ObservationLocation>
            <AstroCoordSystem id="UTC-FK5-GEO"/>
            <AstroCoords coord_system_id="UTC-FK5-GEO">
              <Time unit="s">
                <TimeInstant>
                  <ISOTime>2022-04-04T11:54:30.2</ISOTime>
                </TimeInstant>
              </Time>
              <Position2D unit="deg">
                <Name1>RA</Name1>
                <Name2>Dec</Name2>
                <Value2>
                  <C1>50.4488</C1>
                  <C2>10.0041</C2>
                </Value2>
                <Error2Radius>0.2</Error2Radius>
              </Position2D>
            </AstroCoords>
          </ObservationLocation>
        </ObsDataLocation>
      </WhereWhen>
      <Why importance="0.7">
        <Inference probability="0.7">
          <Concept>process.variation.burst;em.gamma</Concept>
        </Inference>
      </Why>
      <How>
        <Description>N1e notice, data from ECLAIRs instrument</Description>
        <Reference uri="https://www.svom.eu/en/telescope-eclairs-en"/>
      </How>
    </voe:VOEvent>
    ```
</details>
<br/>

<details>
    <summary>`grm-trigger`</summary>

>     ``` xml

Reviewer: ```suggestion
    ```xml
```

          <Param name="Time_Window_Start"  value="2025-02-07T07:14:06.000Z" ucd="time.start"/>
          <Param name="Time_Window_End"    value="2025-02-07T07:14:10.000Z" ucd="time.end"/>
          <Param name="Lower_Energy_Bound" value="14.23"  unit="keV"        ucd="em.energy"/>
          <Param name="Upper_Energy_Bound" value="31.31"  unit="keV"        ucd="em.energy"/>
          <Param name="Triggered_GRDs"     value="101"                      ucd="instr.param"/>
        </Group>
        <Group name="Satellite_Info">
          <Param name="Attitude_Ra"    value="148.96"  unit="deg"  ucd="instr.param"/>
          <Param name="Attitude_Dec"   value="-17.74"  unit="deg"  ucd="instr.param"/>
          <Param name="Attitude_Roll"  value="194.06"  unit="deg"  ucd="instr.param"/>
        </Group>
      </What>
      <WhereWhen>
        <ObsDataLocation>
          <ObservatoryLocation id="GEOLEO"/>
          <ObservationLocation>
            <AstroCoordSystem id="UTC-ICRS-GEO"/>
            <AstroCoords coord_system_id="UTC-ICRS-GEO">
              <Time unit="s">
                <TimeInstant>

>                   <ISOTime>2025-02-07T07:14:06</ISOTime>

Reviewer 2: ```suggestion
                  <ISOTime>2025-02-07T07:14:06Z</ISOTime>
```
For UTC time

                </TimeInstant>
              </Time>
            </AstroCoords>
          </ObservationLocation>
        </ObsDataLocation>
      </WhereWhen>
      <Why importance="0.61">
        <Inference probability="0.61">

>           <Concept>process.variation.burst;em.gamma</Concept>

Reviewer 2: ```suggestion
          <Concept>process.variation.burst;em.gamma</Concept>
```
Is this ; at right place?

        </Inference>
      </Why>
      <How>
        <Description>N1g notice, data from GRM</Description>
        <Reference uri="https://www.svom.eu/en/grm-gamma-ray-burst-monitor-en/"/>
      </How>
    </voe:VOEvent>
```

</details>
<br/>

<details>
    <summary>`mxt-initial_qf1`</summary>

>     ``` xml

Reviewer: ```suggestion
    ```xml
```

---
handle:
  breadcrumb: SVOM
---

import logo from './logo_svom_black.svg'

# Space-based multi-band astronomical Variable Objects Monitor (SVOM)

<img
  src={logo}
  align="right"
  width="818"
  height="780"
  alt="SVOM logo"
  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
/>

**Launch Date:** June 22, 2024

**Anticipated Start of Science Operations:** February 2025

**End of Operations:** Mission duration of 3 years

**Notices Archive:** [SVOM French Science Center](https://fsc.svom.org/alerts)

> The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://cnes.fr/en/projects/svom). SVOM mission is dedicated to the study of the the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).

Reviewer 2: ```suggestion
The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://cnes.fr/en/projects/svom). SVOM mission is dedicated to the study of the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).
```

 <div className="overflow-table">

| <small> Instruments </small> | <small> Energy Range </small>                                          | <small> Field of View </small> | <small> Localization </small>                               |
| ---------------------------- | ---------------------------------------------------------------------- | ------------------------------ | ----------------------------------------------------------- |
| <small> ECLAIRs </small>     | <small> 4 - 250 keV </small>                                           | <small> 2 ster </small>        | <small> ~10ʹ radius (statistical + systematic) </small>     |
| <small> GRM </small>         | <small> 15 - 5000 keV </small>                                         | <small> 2.6 ster </small>      | <small> ~10 degrees </small>                                |
| <small> MXT </small>         | <small> 0.2 - 10 keV </small>                                          | <small> 1.1° x 1.1° </small>   | <small> ~10-100ʺ radius (statistical + systematic) </small> |

> | <small> VT </small>          | <small> 450 - 650 nm (Blue band) <br/>650 - 1000 nm (Red band)</small> | <small> 26ʹ x 26' </small>     | <small> ~1ʺ radius </small>                                 |

Reviewer: What's with all the `<small>` tags?

Reviewer: Please remove the `<small>` tags.

Author: the <small> tags were added just to make the table prettier when the description is a bit long.

Reviewer 3: @timroland We agree that your formatting does make the tables look better, but for now, we'd like to keep things consistent between different pages.  So please remove the "<small>" tags, and we'll look into updating the templates to make things consistent across the board.
